### ISSUE TITLE
There are too many field type categories

### ISSUE THREAD

**Issue by Shelob9** [CalderaWP/Caldera-Forms#607]
1) Basic
1) Advanced
1) Special

Maybe:
1) Select

At most 4 should survive

**Comment by Desertsnowman** [comment]
@Shelob9 I added a Deprecated for reCaptcha

**Comment by Shelob9** [comment]
@Desertsnowman Why is advanced and deprecated first? Make it:

1) Basic 
2) Advanced
3) Special
4) Deprecated

Make deprecated grayed out or something. Also, don't use the term deprecated. We're trying to make the terminology in the UI <em>less</em> technical, not <em>more</em> technical. Use "Discontinued".

Also, this commit only affects core fields. If an add-on adds a field, then it can use whatever category it wants. Please add validation to field category, if it is in an unapproved category, then re-assign it to special. Do NOT add a filter or any other way to define new categories.

And this code is neither secure, nor properly formatted:
1) You used translation strings without properly escaping them.
2) You used double quotes for strings when double quotes were not called for.
3) You commented out code instead of deleting it. 
4) You did not place spaces between operators.

TL;DR :
1) Make it so there are four categories and four categories only. No matter what an add-on uses. Validation!
2) Order the categories based on likelihood of usage not an arbitrary system.
3) Don't use "Deprecated" use "Discontinued"
4) Escape your translation strings.

**Comment by Shelob9** [comment]
Reopening as this classification system makes no sense. Why are we putting extra emphasis on special and discontinued, while "select" an actually useful classification is not available?

**Comment by Shelob9** [comment]
Reclosing with 	33ca873 	aec4f6b 		f1977af

Note, this puts back in a "Content" section with only one field type, which is bad, but I just opened like 5 issues for field types to add in 1.3.7 that will go there.
